src/EVault/modules/Liquidation.sol

```diff
@@ -174,7 +174,9 @@ abstract contract LiquidationModule is ILiquidation, Base, BalanceUtils, Liquidi
 
         // Handle repay: liquidator takes on violator's debt:
 
-        transferBorrow(vaultCache, liqCache.violator, liqCache.liquidator, liqCache.repay);
+        if (liqCache.repay.toUint() > 0) {
```

Comment:
A comment in the github ticket says:

> Another reason to skip transferBorrow is in the case where the violator's debt is dust. In such case, the liquidator would pull the violator's debt

But I don't understand... Transferring dust is legitimate, and required to fully extract any debt before it becomes bad. So, I think this is correct.